source/addComment.js

```diff
@@ -1,3 +1,23 @@
 $(document).ready(function(){
-  // add your code here
-})
+  $('#new_comment_button').on('click', function() {
+
+    $("#new_comment_button").hide();
+
+    $('body').append('<form id="new_comment" name="new_comment" style= "display: block" ></form>')
+
+    $('#new_comment').append('<textarea form="new_comment" id="text" placeholder="Your comment here...">')
+    $('#new_comment').append('<input type="text" id="authorName" placeholder="author" />')
+    $('#new_comment').append('<input type="submit" id="comment_submit" value="Submit Comment!" /> ')
```

Comment:
This is a totally valid approach, but since you are adding a lot of markup, consider writing more HTML and just `.hide()` and `.show()` it with jQuery.